channels/fr.m3u

```diff
@@ -237,8 +233,8 @@ https://dcunilive47-lh.akamaihd.net/i/dclive_1@739146/master.m3u8
 https://alchimie-humanity-1-fr.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KTO.fr" tvg-name="KTO" tvg-country="FR;UK;PT;ES;DK;NL;BE;CH" tvg-language="French" tvg-logo="https://i.imgur.com/LvbXFUy.jpg" group-title="",KTO (404p)
 https://livehdkto-lh.akamaihd.net/i/LiveStream_1@178944/master.m3u8
-#EXTINF:-1 tvg-id="LEquipe.fr" tvg-name="L'Équipe" tvg-country="FR" tvg-language="French" tvg-logo="https://i.ibb.co/bvhBK1y/LEquipe-21.png" group-title="",L'Équipe (576p)
-http://cdn.webtv4.cdnfr.orange.fr/hs/HALO3/hls/equipe21-850463/hls/index.m3u8
+#EXTINF:-1 tvg-id="LEquipe.fr" tvg-name="L'Équipe" tvg-country="FR" tvg-language="French" tvg-logo="https://i.ibb.co/bvhBK1y/LEquipe-21.png" group-title="",L'Équipe [Geo-blocked] (576p)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x2lefik
```

Comment:
query-streamlink.lanesh4d0w.repl.co is down

Reply:
All good now.

Reply:
For me, the error code just changed to 500.
![image](https://user-images.githubusercontent.com/7253922/120471974-0923d080-c3ae-11eb-9048-7fb35eff9213.png)
Or should that be the case if the source is geo-blocked?

Reply:
Well I think that it won't work since it's the IP of the server that does the request. Taking it off.